### DISCUSSION TITLE
In Option Stores, If getters function receive state argument, then can't  use this to find other getters

### DISCUSSION THREAD

**Discussion by wenye123** [vuejs/pinia#2133] [Help and Questions] [answered]
### Reproduction

none

### Steps to reproduce the bug

![image](https://user-images.githubusercontent.com/13810355/230881691-7b8a5fe7-7ca4-44a5-a46d-ceb96fd54f3c.png)
![image](https://user-images.githubusercontent.com/13810355/230881972-5c9b3294-25e1-4a85-bba2-e71c76327a3e.png)

typescript version: 4.8.4

### Expected behavior

Even if state argument exists, we still have to find other getters;

### Actual behavior

can't find other getters

**Comment by posva** [comment, marked as answer]
You need to explicitly add the return type of `info2` as noted in docs: https://pinia.vuejs.org/core-concepts/getters.html#accessing-other-getters
